apps/mobile/mobile-app/components/ui/gluestack-ui-provider/index.tsx

```
@@ -1,38 +1,39 @@
+	mode?: ModeType
+	children?: React.ReactNode
+	style?: ViewProps['style']
 }) {
-  const { colorScheme, setColorScheme } = useColorScheme();
+	const { colorScheme, setColorScheme } = useColorScheme()
 
-  useEffect(() => {
-    setColorScheme(mode);
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [mode]);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: setColorScheme

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)